### ISSUE TITLE
npm install fails with node-gyp

### ISSUE THREAD

**Issue by ax3l** [vmolsa/webrtc-native#25]
Before starting a project that depends on `webrtc-native`, I tried to use one of the tests `test/videoStream.js` as a new `index.js`.
I changed line 1 to write `var WebRTC = require('webrtc-native');` and created a `package.json` as usual.

When running `export BUILD_WEBRTC=true && npm install webrtc-native --save` as instructed in the readme, the install fails with:
```
> webrtc-native@1.2.2 install <myDir>/webrtc-native-test/node_modules/webrtc-native
> node scripts/install.js

Building module...
module.js:328
    throw err;
    ^

Error: Cannot find module '<myDir>/webrtc-native-test/node_modules/webrtc-native/node_modules/node-gyp/bin/node-gyp.js'
[...]
```

since the [scripts/install.js](https://github.com/vmolsa/webrtc-native/blob/b04cf059fd41cb41b2192c9dcb7209def679b805/scripts/install.js#L21) looks in the wrong sub-dir for the `node-gyp` module.

It should not look for it in
```
<myDir>/webrtc-native-test/node_modules/webrtc-native/node_modules/node-gyp/
```
but rather in
```
<myDir>/webrtc-native-test/node_modules/node-gyp/
```

**Comment by vmolsa** [comment]
Hi,

When you are building from source you must clone repository before start [building](https://github.com/vmolsa/webrtc-native#build-from-source).

**Comment by ax3l** [comment]
Aehm, that's what I did :)

New project using `webrtc-native`:
```bash
$ cat index.js
var WebRTC = require('webrtc-native');
# ...

$ npm install webrtc-native --save
> webrtc-native@1.2.8 install [...]/node_modules/webrtc-native
> node scripts/install.js
Downloading module...
[...]/node_modules/webrtc-native/scripts/install.js:86
        throw new Error('prebuilt module not found. See the instructions from https://github.com/vmolsa/webrtc-native#build-from-source for building module from source.');

$ export BUILD_WEBRTC=true
$ npm install webrtc-native --save
```

Actually it should only be `npm install webrtc-native --save` as listed under [install](https://github.com/vmolsa/webrtc-native#install)?

When trying to build from source I did run into #24.

**Comment by vmolsa** [comment]
If you want to install prebuilt modules then ```npm install webrtc-native --save``` works but if you want to build it from source you need to clone it first and build it from that directory.

**Comment by ax3l** [comment]
That is what I try to do: I would like to avoid building from source and want to use the prebuild modules but I get the errors shown above when using `npm install webrtc-native --save`.

Did I miss some download of the pre-build modules so `npm install webrtc-native --save` can succeed?

**Comment by bfishman** [comment]
I do not believe pre-built modules are currently available. Hopefully they will be made available, as multiple downloads of chromium across various failed build attempts are rapidly consuming my monthly data cap! In another issue, vmolsa mentioned his build server is offline at this time, however.

If I am able to build, I will upload my binaries (x64 Ubuntu 15.10)
